Sources/CNIOWindows/shim.c

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#include "CNIOWindows.h"
```

Comment:
There's nothing magic about the `shim.c` file, so we can just remove it.

Reply:
This is going to be needed soon (the sendmmsg Abe recvmmsg functions will reside here.  I suppose we can delay adding it.

Reply:
Ok, we can leave it then.